src/main/java/org/grizz/factory/PlanetFactory.java

```diff
@@ -0,0 +1,28 @@
+package org.grizz.factory;
+
+import com.google.common.collect.Lists;
+import org.grizz.model.Building;
+import org.grizz.model.Planet;
+import org.grizz.model.User;
+import org.grizz.model.enummerations.BuildingType;
+import org.springframework.stereotype.Component;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Component
+public class PlanetFactory {
+
+
+    private static String DEFAULT_NAME = "Planet Mother";
```

Comment:
Please don't use any of human language on backend side. Especially hardcoded. Prefer using key-values like in properties files. It is much easier to introduce i18n on frontend side then